setup.py

```diff
@@ -17,17 +17,17 @@
     packages=find_packages(exclude=['sandbox*', 'tests*']),
     include_package_data=True,
     install_requires=[
-        'django>=1.11,<2.2',
+        'django>=1.11,<2.2.8',
```

Comment:
Please change this to `django>=1.11,<2.3`. Django 2.2.8 has security vulnerabilities, so we shouldn't pin the minor version.